src/constants/regexp.ts

```diff
@@ -0,0 +1 @@
+export const DIGIT_RE = new RegExp(/^\d+$/)
```

Comment:
I would name it `REGEXP_DIGIT`. Common part should be in prefix

Reply:
I've removed this constant and added a misc `isNumeric` utility